packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -815,11 +815,35 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 		}
 
 		toggleMoreComponents () {
+			if (!this.state.isFirstReadInfoComponents) {
+				this.focusMoreButton();
+			}
+
 			this.setState((prevState) => {
 				return {
 					showMoreComponents: !prevState.showMoreComponents
 				};
 			});
+
+		}
+
+		focusMoreButton = () => {
+			if (!this.props.visible) {
+				return;
+			}
+
+			// Readout 'more' or 'back' button explicitly.
+			let selectedButton = Spotlight.getCurrent();
+			if (selectedButton === this.mediaControlsNode.querySelector(`.${css.moreButton}`)) {
```

Comment:
Why did you use `querySelector` instead of `React Ref`?

Reply:
This is the code that @ryanjduffy , @viodragon2 wrote.
I will change this code to use React ref instead of querySelector. 
Please let me know if you have any other comments.